active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+Please view the guidance for [Adjust layout and fonts, and support RTL](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/adjust-layout-and-fonts--and-support-rtl) for more information about text localization in your notification.
+
+# Remarks
+## Usage Recommendations
+
+### When to show an info bar?
+An InfoBar should be shown when the state of the application is different from typical, expected functionality or when the user **needs** to acknowledge or potentially react to the presented information.
+
+Recommended patterns for error notifications where the application is in a negatively altered state
+  - Example: Internet connectivity is required for the application to function and is not present
+    - An error info bar should remain in view until the connection is restored if most functionality is unavailable.
+    - If some app functionality is possible, an error info bar should appear and able to be dismissed by the user.
+    - If/when the internet is restored the existing notification should update if it exists, and a new one should be created if the previous one had been dismissed. THe new/updated success-styled InfoBar should inform the user that "Internet is reconnected" so that they are aware app functionality is restored.
+  - Example: The user's subscription to your application has expired and many features are deactivated
+    - An error info bar could remain in view with a potential action button to resubscribe.
+    - Another option is to allow the user to dismiss a default-style InfoBar depending on how vital a subscription is to your application.
+
+Recommended patterns for informational notifications where the user needs to view or react to essential information
+ - Example: Terms and Conditions of the application have been updated and need to be acknowledged for continued use
+   - A default info bar could appear notifying the user the T&C have updated and link to the contents. The user acknowledges these updated T&C through dismissing the notification.
+ - Example: A virus scan that was running in the background of your application has completed
+   - A success-styled info bar could appear notifying the user the scan has completed and direct them to view the results.
+
+ Note: Users should **always** have the option to dismiss informational notifications.
+
+# API Notes
+
+### Notable Properties  
+
+| Name | Description |
+|:-:|:--|
+| IsOpen| Gets or sets a value that determines the visibility of the InfoBar. By default, is set to false. |
+| Severity | Gets or sets a value that indicates the  color and icon to style the InfoBar. It will also define the assistive technology settings. |
+| IsUserDismissable| Gets or sets a boolean that indicates whether the user will be able to dismiss the InfoBar.
+
+
+### Events  
+| Name | Description |
+|:-:|:--|
+| CloseButtonClick | Occurs after the close button has been tapped. |
```

Comment:
"tapped" => "clicked"  ("Tap" is a touch term, and you can use a mouse or keyboard on a button.)